apps/dashboard/lib/trpc/routers/key/updateRootKeyName.ts

```
@@ -3,14 +3,14 @@ import { db, eq, schema } from "@/lib/db";
 import { TRPCError } from "@trpc/server";
 import { z } from "zod";
 import { auth, t } from "../../trpc";
-
+import { auth, t } from "../../trpc";
```

Biome

[error] Shouldn't redeclare 'auth'. Consider to delete it or rename it.

'auth' is defined here:


(lint/suspicious/noRedeclare)